.github/workflows/build-test-ship.yaml

```diff
@@ -110,10 +113,10 @@ jobs:
               chartchange="false"
             fi
           done
-          echo "::set-output name=appversion::${APPVERSION}"
-          echo "::set-output name=chartversion::${CHARTVERSION}"
-          echo "::set-output name=appchange::${appchange}"
-          echo "::set-output name=chartchange::${chartchange}"
+          echo "appversion=${APPVERSION}" >> $GITHUB_OUTPUT
+          echo "chartversion=${CHARTVERSION}" >> $GITHUB_OUTPUT
+          echo "appchange=${appchange}" >> $GITHUB_OUTPUT
+          echo "chartchange=${chartchange}" >> $GITHUB_OUTPUT
```

Comment:
The `set-output` command [is deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/).